cli/cli_util.c

```diff
@@ -115,10 +120,30 @@ kvdb_compact_request(const char *kvdb_home, const char *request_type, u32 timeou
     uint   sleep_secs = 2;
     char **namev;
     size_t namec;
+    size_t kvdb_paramc = 0;
+    const char *kvdb_paramv[1] = { 0 };
+    size_t kvs_paramc = 0;
+    const char *kvs_paramv[1] = { 0 };
+
+
+    /* Full compactions need a few extra params.
+     */
+    if (request == req_compact_full) {
+        kvdb_paramv[kvdb_paramc++] = "csched_full_compact=true";
+        assert(kvdb_paramc <= NELEM(kvdb_paramv));
+        kvs_paramv[kvs_paramc++] = "cn_close_wait=true";
+        assert(kvs_paramc <= NELEM(kvs_paramv));
+    }
 
-    err = hse_kvdb_open(kvdb_home, 0, NULL, &handle);
+    err = hse_kvdb_open(kvdb_home, kvdb_paramc, kvdb_paramv, &handle);
     if (err) {
         handle = 0;
+        if (hse_err_to_errno(err) == EBUSY && request == req_compact_full) {
+            fprintf(stderr, "Unable to start full compaction KVDB because "
```

Comment:
Something is off with the english here. What is our policy on periods at the end?

Reply:
Some error messages use periods some don't.

Message could be:
- Unable to start full compaction because KVDB is open by another process.

Or fallback to the slightly less informative:
- Failed to open the KVDB (%s): %s\n", kvdb_home, buf);   // where buf contains hse_strerror

Reply:
Second seems consistent with what we usually do.